packages/es-components/src/components/containers/popoverTest/PopoverTest.js

```diff
@@ -0,0 +1,273 @@
+            name={name}
+            trigger={triggerButton}
+            placement={placement}
+            transitionIn={this.state.isOpen}
+            onHide={this.hidePopover}
+            arrowSize={arrowSize}
+            hasTitle={hasTitle}
+            theme={theme}
+          >
+            <PopoverContainer
+              role="dialog"
+              ref={elem => {
+                this.contentRef = elem;
+              }}
+              tabIndex={-1}
+            >
+              <PopoverHeader
+                hasTitle={hasTitle}
+                ref={elem => {
+                  this.headerRef = elem;
+                }}
+                tabIndex={-1}
+              >
+                {hasTitle ? <TitleBar>{title}</TitleBar> : ''}
+                {hasAltCloseButton ? altCloseButton : ''}
+              </PopoverHeader>
+
+              <PopoverBody hasAltCloseWithNoTitle={hasAltCloseWithNoTitle}>
+                <PopoverContent showCloseButton={showCloseButton}>
+                  {content}
+                </PopoverContent>
+                {showCloseButton ? closeButton : ''}
+              </PopoverBody>
+            </PopoverContainer>
+          </Popup>
+        </Container>
+      </ThemeProvider>
+    );
+  }
+}
+
+PopoverTest.propTypes = {
+  /** The name of the popover. Used for differentiating popovers */
+  name: PropTypes.string,
+  /** The link content which activates the popover */
+  children: PropTypes.node.isRequired,
+  /** The text displayed in the popover title section */
+  title: PropTypes.string,
+  /** The content displayed in the popover body */
+  content: PropTypes.node,
+  /** The placement of the popover in relation to the link */
+  placement: PropTypes.oneOf(['top', 'right', 'bottom', 'left']),
+  /** Prevents popover from closing when clicked outside of it */
+  keepOpen: PropTypes.bool,
+  /** The size of the arrow on the popover box */
+  arrowSize: PropTypes.oneOf(['sm', 'lg', 'none', 'default']),
+  /** Prevents the popover from changing position when the window is resized/scrolled */
+  disablePopoverFlipping: PropTypes.bool,
+  /** Display a close button in the bottom right of the popover body */
+  hasCloseButton: PropTypes.bool,
+  /** Display a close ('x') button */
+  hasAltCloseButton: PropTypes.bool,
+  /** Function to toggle display of popover */
+  onToggle: PropTypes.func,
+  /** Displays or hides the popover */
+  showPopover: PropTypes.bool,
+  /** Hide underline from link. Useful for children like Icons */
+  suppressUnderline: PropTypes.bool,
+  /** The button style of the popover link */
+  buttonStyle: PropTypes.string,
+  /** Sets the link to use a text rather than a button style **/
+  isLinkButton: PropTypes.bool,
+  /** Sets the aria-label attribute to allow for textless buttons **/
+  ariaLabel: PropTypes.string,
+  /**
+   * Theme object used by the ThemeProvider,
+   * automatically passed by any parent component using a ThemeProvider
+   */
+  theme: PropTypes.object
```

Comment:
No, theme will *not* be automatically passed in by a parent `ThemeProvider`.

Reply:
That was copy-pasted from the `PopoverLink` props. I will refactor a lot of the wording once I finish the functionality of the popovers themselves.